sustainability_employee_commuting/models/hr_employee.py

```diff
@@ -70,3 +86,27 @@ def _calculate_commuting_carbon(self, date):
             total_uncertainty_value,
             total_carbon_details,
         )
+
+    @api.depends(
+        "monday_location_id",
+        "tuesday_location_id",
+        "wednesday_location_id",
+        "thursday_location_id",
+        "friday_location_id",
+        "saturday_location_id",
+        "sunday_location_id",
+        "exceptional_location_id",
```

Comment:
Isn't this possible ?
```suggestion
        **DAYS
```

Reply:
yes but like this
`    @api.depends(
        *DAYS,
        "exceptional_location_id",
    )`